### PR TITLE
Add sponsors and enhance sponsors section

### PR DESCRIPTION
**Pull request by AlexVelezLl**
image

### CODE REVIEW

File: data/sponsors.json [hunk 1]
```diff
@@ -1,23 +1,44 @@
...
-              "name":"Universidad de Especialidades Espíritu Santo",
-              "link": "https://uees.edu.ec",
-              "logo": "Logo-UEES-2018---Vertical.png"   
-          }
-      ]
-  }
-
+    {
+        "category_name": "Sponsors Gold",
+        "category_icon": "gold_medal.png",
```
Comment (practicatto) [inline data/sponsors.json]: I don't think the medals fit with the devfest style but if no one else thinks the same go ahead

Comment (AlexVelezLl) [reply data/sponsors.json]: If you have a better suggestion, youre welcome :). WIthout any icon/medal there, this section looked a bit poor.

Comment (practicatto) [reply data/sponsors.json]: image

Yes, i agree! What do you think? :)

Comment (AlexVelezLl) [reply data/sponsors.json]: Yees it looks better, thanks!!

Comment (practicatto) [reply data/sponsors.json]: I just pushed a commit, any feedback is welcome :)

**Pull request merged**